### ISSUE TITLE
This synthetic event is reused for performance reasons

### ISSUE THREAD

**Issue by yann86** [igordanchenko/yet-another-react-lightbox#45]
### Describe the bug

When I use yet-another-react-lightbox with react 16 I can't swipe between photos, I get an error related to event pooling how had been removed in react 17.

<img width="1294" alt="image" src="https://user-images.githubusercontent.com/1409790/206301559-7e059399-7faa-4b2c-b9b3-0e40e896d2e6.png">

### Expected behavior

I Expect a swipe action with react 16 because it's included in package.json has peerDependency

### How to reproduce

https://codesandbox.io/s/yet-another-react-lightbox-examples-forked-ph4z0f?file=/src/index.js

**Comment by igordanchenko** [comment]
Thank you for reporting this. Please let me know if v2.2.3 addresses your issue.

**Comment by yann86** [comment]
> Thank you for reporting this. Please let me know if v2.2.3 addresses your issue.

Yes with v2.2.3 swipe works as expected. Thank you for the quick update
